build-system/babel-plugins/babel-plugin-jsx-style-object/index.js

```diff
@@ -123,6 +123,43 @@ module.exports = function (babel) {
     return expr;
   }
 
+  /** @param {babel.NodePath} path */
+  function replaceExpression(path) {
+    if (path.isLogicalExpression()) {
+      if (path.node.operator === '&&' || path.node.operator === '||') {
+        replaceExpression(path.get('right'));
+      }
+      return;
+    }
+    if (path.isConditionalExpression()) {
+      replaceExpression(path.get('consequent'));
+      replaceExpression(path.get('alternate'));
+      return;
+    }
+    if (!path.isObjectExpression()) {
+      return;
```

Comment:
Should we throw instead?

Reply:
Strings and results that return strings are valid.